src/sidebar/test/media-embedder-test.js

```diff
@@ -50,14 +70,12 @@ describe('media-embedder', function () {
 
       mediaEmbedder.replaceLinksWithEmbeds(element);
 
-      assert.equal(element.childElementCount, 1);
-      assert.equal(element.children[0].tagName, 'IFRAME', url);
-      // queryString's #stringify sorts keys, so resulting query string
-      // will be reliably as follows, regardless of original ordering
-      // Note also `v` param is handled elsewhere and is not "allowed" in
-      // queryString.
       assert.equal(
-        element.children[0].src,
+        embedUrl(element),
+        // queryString's #stringify sorts keys, so resulting query string
+        // will be reliably as follows, regardless of original ordering
+        // Note also `v` param is handled elsewhere and is not "allowed" in
```

Comment:
Is `v` param the video ID, that is now part of the path?

Reply:
This comment was here previously. I just moved it around in this PR as part of eliminating some duplication in the tests.